src/features/click-to-play.js

```diff
@@ -834,7 +840,7 @@ function makeShareFeedbackRow () {
 }
 
 /* FB login replacement button, with hover text */
-function makeLoginButton (buttonText, mode, hoverTextTitle, hoverTextBody, icon, originalElement) {
```

Comment:
Note that it looks like `hoverTextTitle` had been moved to `styles` but it remained here as a method argument, which seems like a bug?